Backpack/src/main/java/net/skyscanner/backpack/text/BpkTextField.kt

```diff
@@ -102,21 +103,36 @@ open class BpkTextField @JvmOverloads constructor(
     }
 
     this.iconTintColor = iconColor
-    setTextColor(textColor)
+    setTextColor(
+      colorStateList(
```

Comment:
Here we're just using a builder function introduced recently, configuring the colours in all states.